### ISSUE TITLE
Video combiner not working

### ISSUE THREAD

**Issue by Sirdarcy69** [inbasic/media-converter#20]
Hello, I tried to combine a video stream and an audio stream but the media converter crashes immediately. Here is the error log screen shot:
![2017-03-16_15-13-59](https://cloud.githubusercontent.com/assets/26455228/23987510/8d557706-0a5c-11e7-896b-6671f771b9ba.jpg)

Thanks,

S

**Comment by inbasic** [comment]
Can you share the files here for testing? You can zip them and drop them here

**Comment by Sirdarcy69** [comment]
It won't let me attach the files as they are too big (total of 40MB)

**Comment by inbasic** [comment]
Can you share the YouTube video link then?

**Comment by Sirdarcy69** [comment]
Here is the link: https://www.youtube.com/watch?v=clX7SBsAb8A&t=4s
I downloaded the video stream and audio stream separately using the Firefox add-on "You Tube Video and Audio Downloader" so I can get the 480p resolution.

**Comment by inbasic** [comment]
What is the filename of both audio and video files?

Can you replace them with video.mp4 and audio.m4a and see if it helps?

**Comment by Sirdarcy69** [comment]
I changed the names as suggested but it crashed all the same

**Comment by inbasic** [comment]
are you sure both files are downloaded correctly? make sure their size is not zero. I am going to try to reproduce this.

**Comment by inbasic** [comment]
do you have issues with other conversions tabs too? like converting to mp3?

**Comment by Sirdarcy69** [comment]
I checked both files and they work perfectly. So it's not the video/audio files.  
Mp3 conversion also crashes the converter. here is the log:
![2017-03-17_9-23-30](https://cloud.githubusercontent.com/assets/26455228/24026474/fd7c4f18-0af3-11e7-9348-582d650e5210.jpg)

I just tried to upload the audio file as a .zip but get the message: "We don’t support that file type. with a PNG, GIF, JPG, DOCX, PPTX, XLSX, TXT, PDF, or ZIP. " ?????

**Comment by inbasic** [comment]
so most likely the downloaded FFmpeg is corrupted. Try to get a fresh copy and set the path to the new executable in the settings tab.

**Comment by Sirdarcy69** [comment]
Yep, that was the problem indeed! The FFmpeg exe had only downloaded partially. An idea might be to add a progress bar and a way of checking that it has been downloaded properly rather than just the hourglass. Just a thought of course. Thanks a lot for your help. I really appreciate it!

**Comment by AudioArchitect-R** [comment]
I have the same problem.  This is what I got when installing the native client:

user@Name ~ $ ./install.sh
NodeJS status = 1
./install.sh: line 8: node: command not found
Installer is using the attached NodeJS
 -> Root directory is /home/user
 -> Creating a directory at /home/user/.config/google-chrome/NativeMessagingHosts
 -> Chrome Browser is supported
 -> Creating a directory at /home/user/.config/chromium/NativeMessagingHosts
 -> Chromium Browser is supported
 -> Creating a directory at /home/user/.config/vivaldi/NativeMessagingHosts
 -> Vivaldi Browser is supported
 -> Creating a directory at /home/user/.mozilla/native-messaging-hosts
 -> Firefox Browser is supported
 -> Creating a directory at /home/user/com.add0n.node
 => Native Host is installed in /home/user/com.add0n.node


>>> Application is ready to use <<<


I'm thinking a "command not found" error might do it, no?

TIA

**Comment by andy-portmen** [comment]
@AudioArchitect-R this seems like there is a problem in my native client. What is the output of `which node` when executed in the console? What is your OS?

**Comment by FanOfMiku39** [comment]
OK, haven't used MC&M all that much until recently, wanting to use the video combiner since a firefox addon, youtube video and audio downloader [dev. edition] breaks/separates the files when you want highest resolution. When I drag the two files to the drop box [files renamed to videoHMMM and audioHMMM] theres no issue; MC&M seems to be doing the job. But when I play the combined file, it is only the audio file there under a different file name [a -1 is added at end of file]. Am I missing a sequence or something? will attach the two files.
ok, cannot attach; files are not supported

**Comment by FanOfMiku39** [comment]
Ok, the audio extractor did indeed work with that sample video
[mov_bbb.zip](https://github.com/inbasic/media-converter/files/2178451/mov_bbb.zip)

**Comment by inbasic** [comment]
> is there a size limit for MC&M

Nope!

> Hm, cannot put files bigger than 10MB here and it is zipped

Use Dropbox or another file sharing service and share the link to the ZIP file then

**Comment by FanOfMiku39** [comment]
Here are OneDrive links to the audio and video that I'm trying to combine:

https://1drv.ms/u/s!AujEZY6sbIrZhXU_HmNJ9tBI3A2x

https://1drv.ms/u/s!AujEZY6sbIrZhXa9MBw5cvz4lQO5
